packages/grid/x-data-grid/src/components/cell/GridCell.tsx

```diff
@@ -341,16 +342,18 @@ const GridCell = React.forwardRef<HTMLDivElement, GridCellProps>((props, ref) =>
         opacity: 0,
         width: 0,
         border: 0,
+        ...styleProp,
```

Comment:
It's ok to skip here, the cell is supposed to be invisible so styles don't matter, we only want to ensure the invisible style applies.